src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -693,6 +693,12 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     // TODO: Change phrasing to use better english once we can tag settings, right now it's kept as history instead of historical so that the setting shows up when the user searches for history
     layout.addIntInput("Max number of history messages to load on connect",
                        s.twitchMessageHistoryLimit, 10, 800, 10);
+    layout.addIntInput(
+        "Max number of messages for user chat logs (requires restart) ",
+        s.twitchMessageLogLimit, 1000, 100000, 1000);
+    layout.addIntInput(
+        "Max number of on-screen messages in a channel (requires restart) ",
+        s.twitchMessageOnScreenLimit, 1000, 100000, 1000);
```

Comment:
Update these labels once the variable names have been changed, also remove the space suffix